examples/applications/semantic-search/semantic_search_quora_elasticsearch.py

```diff
@@ -67,13 +72,15 @@
                 },
                 "question_vector": {
                   "type": "dense_vector",
-                  "dims": 768
+                  "dims": 768,
+                  "index": True,
+                  "similarity": "cosine"
                 }
               }
             }
         }
 
-        es.indices.create(index='quora', body=es_index, ignore=[400])
```

Comment:
This argument is deprecated and I would argue that a bad request should fail hard here anyways.